### PR TITLE
chore(release): adjust BPMN 2.0 examples

### PR DESCRIPTION
**Pull request by tasso94**
* usertask/task-form-generated
* usertask/task-form-external-jsf
* usertask/task-form-embedded-serialized-java-object
* usertask/task-form-embedded-react
* usertask/task-form-embedded-bpmn-events
* usertask/task-form-embedded
* usertask/task-assignment-email
* usertask/task-form-embedded-json-variables
* usertask/task-camunda-forms
* wildfly/jackson-annotations
* authentication/basic
  * fixes CAM-13366
* scripttask/xslt-scripttask
* scripttask/xquery-scripttask
* startevent/soap-cxf-server-start
* startevent/message-start
* servicetask/soap-service
* servicetask/soap-cxf-service
* servicetask/service-invocation-synchronous
* servicetask/service-invocation-asynchronous
* servicetask/rest-service

related to CAM-13908

### CODE REVIEW

**Review by mboskamp — APPROVED** [review]
Well done on the reformatting! I have added only a very few suggestions. Feel free to merge if you disagree. :)

File: scripttask/xslt-scripttask/src/test/resources/expected_result.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <users>
   <user>
-      <name>Stefan</name>
-      <familyname>Hentschel</familyname>
-      <birthday>18.11.1988</birthday>
+    <name>Stefan</name>
```
[the remark this answers is not in the document]
Comment (tasso94) [reply scripttask/xslt-scripttask/src/test/resources/expected_result.xml:4]: 👍

File: scripttask/xquery-scripttask/src/main/resources/org/camunda/bpm/example/xqueryexample/example_names.xml [hunk 2]
```diff
@@ -2,12 +2,12 @@
 <names xmlns="http://my.namespace/v1">
   <person>
     <id>1</id>
-   	<firstName>Thomas</firstName>
-   	<lastName>Skjolberg</lastName>
+    <firstName>Thomas</firstName>
```
[the remark this answers is not in the document]
Comment (tasso94) [reply scripttask/xquery-scripttask/src/main/resources/org/camunda/bpm/example/xqueryexample/example_names.xml:5]: 👍

File: scripttask/xquery-scripttask/src/test/resources/expected_result.xml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
-	<person>
-		<id>2</id>
-		<firstName>Kermit</firstName>
-		<lastName>Frog</lastName>
-		<skill>Talk</skill>
-		<skill>Smile</skill>
-	</person>
+  <person>
+    <id>1</id>
+    <firstName>Thomas</firstName>
```
[the remark this answers is not in the document]
Comment (tasso94) [reply scripttask/xquery-scripttask/src/test/resources/expected_result.xml:4]: 👍

File: startevent/message-start/pom.xml [hunk 4]
```diff
@@ -11,6 +11,10 @@
 
   <properties>
     <version.camunda>7.16.0</version.camunda>
+    <version.junit>4.13.2</version.junit>
+    <version.h2>1.4.190</version.h2>
```
[the remark this answers is not in the document]
Comment (tasso94) [reply startevent/message-start/pom.xml]: 👍

**Review by mboskamp — APPROVED** [review]

**Pull request merged**